### PR TITLE
Pass bucket to model-plugin-deployer directly

### PR DESCRIPTION
**Pull request by christophermaier**
Same principle as https://github.com/grapl-security/grapl/pull/1044

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -382,6 +383,14 @@ services:
         condition: service_healthy
       provisioner:
         condition: service_completed_successfully
+      pulumi:
+        # We must wait until Pulumi has created the buckets to know what their names are
+        condition: service_completed_successfully
+    volumes:
+      - type: volume
+        source: pulumi_outputs
+        target: /pulumi-outputs
```
Comment (wimax-grapl) [inline docker-compose.yml:392]: (Not a must-do, but I sort of would prefer if target were `/mnt/pulumi-outputs` so it's more obvious this is a volume). However, I recognize I'm asking to reinvent the existing wheel - more something to consider in the future.

File: docker-compose.yml [hunk 2]
```diff
@@ -359,21 +359,22 @@ services:
 
   model-plugin-deployer:
     image: grapl/model-plugin-deployer:${TAG:-latest}
-    command: |
-      /bin/sh -c '
-        . venv/bin/activate &&
-        cd /home/grapl/app &&
+    entrypoint: ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
```
Comment (wimax-grapl) [inline docker-compose.yml:362]: Would it be better to specify the entrypoint in the image?  (This should probably be the entrypoint for everything so we can stop doing && stuff)

**Review by wimax-grapl — APPROVED** [review]

**Review by jgrillo-grapl — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (christophermaier) [reply docker-compose.yml:362]: As for baking this into the images: that's not actually a blanket assumption that we can (or even should) make; it all depends on the nature of the container. The fact that we're having to do all this "extra" work in docker-compose is a bit unique, and not something I'd want to enshrine in our containers.

As for the `&&` stuff, yes, this should ultimately be the entrypoint we use in our docker-compose services that are currently doing `/bin/bash -c 'blah && blah && blah`. I'm updating that as I'm going along with the various services I'm updating.

File: docker-compose.yml (hunk 1 above)
Comment (christophermaier) [reply docker-compose.yml:392]: I could take another pass through everything and do that, on a separate PR.

**Pull request merged**